Velaptor/OpenGL/GLWindow.cs

```diff
@@ -249,23 +283,77 @@ public async Task ShowAsync(Action? afterStart = null, Action? afterUnloadAction
         }
 
         /// <inheritdoc/>
-        public void Close() => this.windowFacade.Close();
+        public void Close() => this.glWindow.Close();
 
         /// <inheritdoc cref="IDisposable.Dispose"/>
         public void Dispose() => Dispose(true);
 
-        /// <summary>
-        /// Invoked when an OpenGL error occurs.
-        /// </summary>
-        private static void GL_GLError(object? sender, GLErrorEventArgs e) => throw new Exception(e.ErrorMessage);
+        private void RunGLWindow()
+        {
+            this.glWindow.Run();
+
+            /*NOTE:
+             * Only dispose of the window here and not in the Dispose() method!!
+             *
+             * This is because this line of code will not be executed until the Window.Run() method
```

Comment:
Grammar
```suggestion
             * This is because that line of code will not be executed until the Window.Run() method
```

Reply:
Refactor to below instead

```suggestion
             * This is because the line of code below will not be executed until the Window.Run() method
```